test/cli-alert/src/index.ts

```diff
@@ -101,7 +101,10 @@ async function waitForConclusion(runID: number): Promise<boolean> {
       }`,
     );
   }
+  console.log('Finished run.');
+}
 
+async function checkJobConclusion(runID: number): Promise<boolean> {
   // Get conclusions of the jobs that failed before
   const smokeTetsJobs = (
```

Comment:
There's a typo here you could consider fixing.
```suggestion
  const smokeTestJobs = (
```